sklearn/decomposition/_nmf.py

```diff
@@ -940,7 +1071,9 @@ def non_negative_factorization(X, W=None, H=None, n_components=None, *,
         - 'cd' is a Coordinate Descent solver that uses Fast Hierarchical
             Alternating Least Squares (Fast HALS).
 
-        - 'mu' is a Multiplicative Update solver.
+        - 'mu' is a Multiplicative Update solver
+            This is the only solver available in
+            the :class:`sklearn.decomposition.MiniBatchNMF` case.
```

Comment:
Might be better to say that it is the only solver available when `batch_size` is not None, since this function can be called outside of MiniBatchNMF.